modules/search/instant-search/lib/dom.js

```diff
@@ -13,6 +13,10 @@ export function getThemeOptions( searchOptions ) {
 			'.searchform input.search-field:not(.jetpack-instant-search__box-input)',
 		].join( ', ' ),
 		filterInputSelector: [ 'a.jetpack-search-filter__link' ],
+		overlayTriggerSelector: [
+			'.jetpack-instant-search__open-overlay-button',
+			'#site-header .header-navigation-wrapper .header-toggles .search-toggle',
```

Comment:
Looks like this one is targeted at Twenty Twenty - shall we add a comment to that effect?